vunit/ui/__init__.py

```diff
@@ -1099,3 +1099,13 @@ def get_simulator_name(self):
         if self._simulator_class is None:
             return None
         return self._simulator_class.name
+
+    def simulator_supports_coverage(self):
+        """
+        Returns True if the simulator supports coverage
```

Comment:
Please, replace `if` with `when`, for consistency with other sources.

Reply:
Done, thanks